perf/run-benchmarks

```diff
@@ -276,15 +185,13 @@ def output_differences(
     return fd_len, sd_len
 
 def run_semgrep(
-    docker: str, corpus: Corpus, variant: SemgrepVariant, include_time: bool
-) -> Tuple[float, bytes]:
+    docker: str, corpus: Corpus, variant: SemgrepVariant, include_time: bool, hard_timeout: int = 0
+) -> Tuple[float, dict]:
     args = []
     common_args = [
-        "--strict",
```

Comment:
do we not want to run with strict?

Reply:
Oh, good catch - I was debugging, so I removed it and forgot to put this back in